app/models/profile_link_site.rb

```diff
@@ -13,7 +13,7 @@
 #
 # rubocop:enable Metrics/LineLength
```

Comment:
Because, remember back to like 3 days ago

When the ProfileLink and LinkedAccount were the same system

Before you, yourself, changed it